src/client/components/element-info/entity-info.js

```diff
@@ -577,12 +577,13 @@ class EntityInfo extends React.Component {
     let targetFromAssoc = (m) => {
       let complete = stage === STAGES.COMPLETED;
       let highlight = !complete;
-      let searchTerm = highlight ? s.name : null;
+      let searchStr = highlight ? s.name : null;
+      let searchTerms = searchStr.split(' ');
```

Comment:
This is probably OK, but users sometimes accidentally type two spaces so `\s+` is safer.